### ISSUE TITLE
get_user_basic_details() not working when using multiple accounts

### ISSUE THREAD

**Issue by OlegSavinov** [iSarabjitDhiman/InstaGPy#7]
When trying to execute get_user_basic_details() form logged_in instance of InstaGPy - function works.
But when instance initialised with sessions from multiple accounts - doesn't. 
I did a little investigation, function get_user_info returns in this case html instead of object. 

Submitting isolated test screenshot, obviously nothing could be changed from cases - they were performed sequentially. 

<img width="1468" alt="Screenshot 2024-03-23 at 18 30 15" src="https://github.com/iSarabjitDhiman/InstaGPy/assets/19290177/5ba08d20-fbd1-46cb-b884-33f8bf49bb0b">

**Comment by iSarabjitDhiman** [comment]
Yes, its quite normal. They block the guest users quite often. If you try to check a Instagram user's profile in an incognito window (guest session) It will not work. If it works, then it will work with the script as well. 
But if you are logged in, you shouldn't be facing any problems.

let me know if you have any other questions, otherwise feel free to close the issue.
Thanks.
